interp/constrexpr.ml

```diff
@@ -17,6 +17,7 @@ open Decl_kinds
 
 (** [constr_expr] is the abstract syntax tree produced by the parser *)
 
+type universe_decl = (lident list, Univ.Constraint.t) gen_universe_decl
```

Comment:
The earliest definition of this type is in ustate, for some reason it was simultaneously (f72a67569ec8cb9160d161699302b67919da5686) created in evd (later moved to ustate) and univdecls.

Reply:
I think the reason this kind of stuff happened is that we had some subtle cyclic dependency mess in the system, but these should be gone by now and enforced once the Dune build is in the CI.

I think the situation is a bit better now, will try to make this type a deprecated alias.

Reply:
Fixed, thanks for the tip.

Reply:
There was indeed quite a few duplication.